src/lib.rs

```diff
@@ -24,7 +24,7 @@
 
 #![no_std]
```

Comment:
I thought we had migrated away from `no_std`. 
This lib is not intended to support no_std compatibility for the time being.

Could we directly remove it? 

Any thougts @vlopes11 @ZER0 ?

Reply:
JubJub is inherently no standard AFAIK. So the code added would have to be `no_std`.

Reply:
I believe we should convert to `std` in another PR that's dedicated to that.

Reply:
KK, can you open a follow-up issue which does that? Since AFAIK we already discusses that and there was no need for having JubJub as a `no_std` crate. Anyway, maybe we can just add a `std` feature.